localstack/testing/snapshots/transformer_utility.py

```diff
@@ -281,6 +281,24 @@ def kinesis_api():
             ),
         ]
 
+    @staticmethod
+    def route53resolver_api():
+        """
+        :return: array with Transformers, for route53resolver api.
+        """
+        return [
+            TransformerUtility.key_value(
+                "SecurityGroupIds", value_replacement="sg-ids", reference_replacement=False
```

Comment:
SecurityGroupIds - this gives a list of security group ids and not string.

Reply:
Not sure I'm following. Can you give an example?

Reply:
Sorry, misunderstood this, you are right. Will update this as well.

Reply:
@dominikschubert
- one doubt how can I add a transformer where i need to convert a list of indices eg: ["ind-1", "ind-2", "ind-3"] to either a string <sample> or ["<sample:1>", "<sample:2>", "<sample:3>"]
- this happens if reference_replacement = True(default) here

Reply:
Not sure I understand. The transformer in general is always just operating on the snapshot `dict` and produces another `dict`.

Would be great if you could just simulate this via one original dict that includes the original SecurityGroupIds so that I can understand the problem you're referring to :thinking:

Reply:
currently going with reference_replacement=False as discussed